group_vars/all.sample

```diff
@@ -1,18 +1,33 @@
 ---
 # Variables here are applicable to all host groups NOT roles
```

Comment:
why did you merge all the files?
IMHO `all.sample` should container `ceph-common`.